nntrainer/src/bn_layer.cpp

```diff
@@ -53,17 +53,17 @@ int BatchNormalizationLayer::initialize() {
   }
 
   Tensor mu =
-    initializeWeight(dim, initializers[static_cast<int>(BNParams::mu)], status);
-  Tensor var = initializeWeight(
-    dim, initializers[static_cast<int>(BNParams::var)], status);
+    getInitializedTensor(dim, initializers[static_cast<int>(BNParams::mu)]);
+  Tensor var =
+    getInitializedTensor(dim, initializers[static_cast<int>(BNParams::var)]);
   NN_RETURN_STATUS();
```

Comment:
NN_RETURN_STATUS could be deleted (same for 61~66)

Reply:
done.